include/jemalloc/internal/jemalloc_internal_types.h.in

```diff
@@ -33,17 +33,18 @@ typedef enum malloc_init_e malloc_init_t;
  *
  * a: arena
  * t: tcache
+ * .: dependent on --with-lg-tcache-limit
  * 0: unused
  * z: zero
  * n: alignment
  *
- * aaaaaaaa aaaatttt tttttttt 0znnnnnn
+ * aaaaaaaa a...tttt tttttttt 0znnnnnn
```

Comment:
I think this needs updating now, given you're allowing a larger range of values in the config param.

Reply:
Done